Content.Shared/Materials/SharedMaterialReclaimerSystem.cs

```diff
@@ -110,6 +112,9 @@ public bool TryStartProcessItem(EntityUid uid, EntityUid item, MaterialReclaimer
             component.NextSound = Timing.CurTime + component.SoundCooldown;
         }
 
+        var reclaimedEvent = new GotReclaimedEvent(uid.ToCoordinates());
+        RaiseLocalEvent(item, ref reclaimedEvent);
```

Comment:
use xformsystem, these methods do a bunch of resolves internally that are slow and not thread-safe.